.github/workflows/release-notes.yml

```diff
@@ -0,0 +1,58 @@
+name: Generate release notes
+
+on:
+  workflow_dispatch:
+    inputs:
+      bc_commit_sha:
+        description: 'Commit SHA for the BC'
+        type: string
+        required: true
+      version:
+        description: 'Version to be released (for example, `9.2.1`)'
+        type: string
+        required: true
+
+env:
+  product: Elastic Agent
+  labels: docs,forwardport-main,release,skip-changelog,Team:Docs
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
why is this needed? IIRC, that's when using the `gh` cli, but in general, I think it should not be needed, but I might be wrong here

Reply:
I added it with the intention that it would be used by elastic-agent-changelog-tool ([docs](https://github.com/elastic/elastic-agent-changelog-tool/blob/main/docs/github-authentication.md)).